NOTICE.txt

```diff
@@ -1,5 +1,5 @@
 Elastic Beats
-Copyright 2014-2018 Elasticsearch BV
+Copyright 2014-2019 Elasticsearch BV
```

Comment:
Should we wait till https://github.com/elastic/beats/pull/9852 is merged and then rebase this PR on `6.x`?

Reply:
Oh no! Totally missed the comments there. Thanks!!